src/test/java/com/github/maracas/roseau/diff/formatter/JsonFormatterTest.java

```diff
@@ -0,0 +1,64 @@
+package com.github.maracas.roseau.diff.formatter;
+
+import java.util.List;
+
+import org.junit.jupiter.api.Test;
+
+import com.github.maracas.roseau.diff.changes.BreakingChange;
+
+import static com.github.maracas.roseau.utils.TestUtils.buildDiff;
+
+class JsonFormatterTest {
+    @Test
+    void method_removed_in_JSON_format() {
+        var v1 = """
+                class A {
+                	public void m1() {}
+                }
+                public class B extends A {
+                	public void m2() {}
+                }""";
+        var v2 = """
+                class A {
+                	void m1() {}
+                }
+                public class B extends A {
+                	public void m2() {}
+                }""";
+
+        List<BreakingChange> breakingChanges = buildDiff(v1, v2);
+
+        String expectedResult = """
+                [{"nature":"DELETION","oldLocation":{"path":"A.java","line":2},"kind":"METHOD_REMOVED","element":"A.m1"}]""";
+
+        JsonFormatter formatter = new JsonFormatter();
+        var result = formatter.format(breakingChanges);
+
+        assert (result.equals(expectedResult));
```

Comment:
It's indeed confusing, but the `assert` statement in Java is used for something completely different that's unrelated to unit testing ;) Unit tests should indeed include assertions, but they are written using specific frameworks, which in the case of Roseau are JUnit 5 (cf. https://junit.org/junit5/docs/5.0.1/api/org/junit/jupiter/api/Assertions.html) and Hamcrest (cf. https://hamcrest.org/JavaHamcrest/tutorial).

You may have a look at https://github.com/alien-tools/roseau/blob/main/src/test/java/com/github/maracas/roseau/api/AnnotationsExtractionTest.java#L22-L23 for some examples.

In your case, the assert statement could look like

```
assertThat(result, is(equalTo(expectedResult));
```

Once these assertions are in place, you should be able to run the tests (e.g. with `mvn test`) and see the tests passing.